mock-api/msw/handlers.ts

```diff
@@ -561,6 +561,13 @@ export const handlers = makeHandlers({
     disk.state = { state: 'detached' }
     return disk
   },
+  instanceEphemeralIpDetach({ path, query }) {
+    const instance = lookup.instance({ ...path, ...query })
+    const ip = db.ephemeralIps.find((eip) => eip.instance_id === instance.id)
```

Comment:
I think an instance can have more than one ephemeral IP, which means you need to match on both the instance ID and the `ip`. This currently works because there's only one.

Reply:
It's possible the docs are incorrect, but https://docs.oxide.computer/guides/managing-instances#_attach_or_detach_external_ip_addresses currently says "An instance may have at most one ephemeral IP, and at most 32 total external IPs." Happy to make this bit of code more robust just in case they end up in a strange state with multiple ephemeral IPs, or in case >1 is legit.

Reply:
Nope, that's correct. Max = 1. I do think you should still match on the IP, though.

https://github.com/oxidecomputer/omicron/blob/895f2800ad96b51c538ae8011df3654d511563c5/nexus/src/app/mod.rs#L104

Reply:
We talked through this and checked the API, and determined that it was best to match the API logic, to simply detach the first ephemeral IP found (which should be the only ephemeral IP present).